src/Nitric.Sdk/Topics/Topic.cs

```diff
@@ -12,42 +12,47 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-using Nitric.Proto.Event.v1;
+using System.Threading.Tasks;
+using Nitric.Proto.Topics.v1;
 using Nitric.Sdk.Common;
 
-namespace Nitric.Sdk.Event
+namespace Nitric.Sdk.Topics
 {
     /// <summary>
     /// Represents a reference to a topic.
     /// </summary>
     public class Topic<T>
     {
-        private readonly EventsClient<T> eventsClient;
+        private readonly TopicsClient<T> TopicsClient;
```

Comment:
As mentioned in another comment, wonder if it's worth cutting out the middleman here and just integrating with the Grpc client directly? If we need to introduce a testing boundary that can be done with an overloaded constructor for supplying a mock client?